src/components/views/Matches.js

```diff
@@ -40,36 +40,42 @@ const Matches = () => {
     useEffect(() => {
         getMatches()
     }, [])
-
+    // for styling: animationDelay={index + 1}
     return (
         <React.Fragment>
             <Header view="game"/>
-            <div className={"game description"}>
-                <h1>You matched with:</h1>
-            </div>
+            <div className="containerMatchList">
+                <div className="matchListHeading">
+                    <h2>Matches</h2>
+                </div>
+                <div className="messageList">
+                    <div className="searchWrap">
+                        <input type="text" placeholder="Search Here" required />
+                        <button className="searchButton">
+                            <i className="fa fa-search"></i>
+                        </button>
+                    </div>
+                </div>
+                <div className="MatchListItem">
+                    {matchedUsers ? matchedUsers.map(user => {
+                        return (
+                            <div>
+                                <MatchListItems
+                                    name={user.username}
+                                    key={user.id}
+                                    active={user.status ? "ONLINE" : ""}
+                                    isOnline={user.status ? "ONLINE" : ""}
+                                    status={user.status ? "ONLINE" : ""}
```

Comment:
Is that all the same and why is alternative empty string and not OFFLINE?

Reply:
I will test and change it